tests/testthat/test-local-timer.R

```diff
@@ -0,0 +1,40 @@
+test_that("local_timer no title", {
+  fun <- function(x) {
+    tmr <- local_timer(title = FALSE)
+    Sys.sleep(0.1)
+    NULL
+  }
+  expect_message(fun(), "^00:00:00.1\\d{2,2}\\s*$")
+})
+
+test_that("local_timer title thing", {
+  fun <- function(x) {
+    tmr <- local_timer(title = "thing")
+    Sys.sleep(0.1)
+    NULL
+  }
+  expect_message(fun(), "^00:00:00.1\\d{2,2} \\[thing\\]\\s*$")
+})
+
+test_that("local_timer title TRUE", {
+  fun <- function(x) {
+    tmr <- local_timer(title = TRUE)
+    Sys.sleep(0.1)
+    NULL
+  }
+  expect_message(fun(), "^00:00:00.1\\d{2,2} \\[.*:.*\\]\\s*$")
```

Comment:
Can the regex be more precise?

```suggestion
  expect_message(fun(), "^00:00:00.1\\d{2,2} \\[.*test-local-timer\\.R:[1-9][0-9]*\\]\\s*$")
```